resources/definitions/fdmprinter.def.json

```diff
@@ -1150,6 +1150,18 @@
                     "minimum_value_warning": "math.cos(wall_transition_angle / 180 * math.pi) * wall_line_width_x",
                     "maximum_value_warning": "10 * math.cos(wall_transition_angle / 180 * math.pi) * wall_line_width_x"
                 },
+                "wall_transition_filter_deviation":
+                {
+                    "label": "Wall Transition Filter Deviation",
+                    "description": "Prevent Wall Transition Filtering from inducing errors on the extrusion width larger than this value.",
+                    "type": "float",
+                    "unit": "mm",
+                    "default_value": 0.1,
+                    "value": ".25 * machine_nozzle_size",
```

Comment:
Do we really need this to be a setting? In what cases would you want to increase or reduce this setting, where that wouldn't be acceptable for a different print? What if we simply hard-code this to 25% nozzle size in CuraEngine, would anyone be bothered by it?

After discussing this with colleagues, we think this solution is best:
* If this is something that can be argued that it really needs to be different for different prints, we'd like to come up with a better name.
* If this is something that is going to be tweaked before the release, we'd like to make this a hidden setting until it is tweaked, to be removed later.
* If 25% of nozzle size is fine for pretty much all cases, we'd like to hard-code it in CuraEngine.

We hope this will make it more clear to use, both for the user and for ourselves in the future.